storybookjs/storybook
# Manager: Fix configuring UI features through URL parameters

Closes #23550



## What I did

Fix unable to configure features (full screen, nav, panel, etc ...) through URL parameters

## How to test

1. Run `yarn start`
2. Open Storybook in your browser
3. Should able to change panel position from bottom to right using URL Parameters (`&panel=right`, `&panel=bottom`, ...)

## Example links 
- http://localhost:6006/?path=/story/example-button--primary&panel=bottom
- http://localhost:6006/?path=/story/example-button--primary&panel=right
- http://localhost:6006/?path=/story/example-button--primary&panel=false

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`



### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
